docs/configuration.md

```diff
@@ -284,7 +284,9 @@ export default class HtmlPage extends React.Component {
 And in your `batfish.config.js` file you'd use [`webpackStaticIgnore`] so the module at `'../app'` doesn't actually get parsed during the static HTML build:
 
 ```js
-{ webpackStaticIgnore: [path.join(__dirname, './src/app')] }
+{
+  webpackStaticIgnore: [path.join(__dirname, './src/app')];
+}
```

Comment:
Looks like there are a lot of unnecessary changes to this doc. Do you have a code editor extension messing with code blocks?